src/rules/at-rule-empty-line-before/index.js

```diff
@@ -6,6 +8,7 @@ import {
   ruleMessages,
   validateOptions,
 } from "../../utils"
+import { cssStatementIsIgnoredAtRule } from "../block-opening-brace-space-before"
```

Comment:
I think we should pull this out of `block-opening-brace-space-before` and into a util called `optionsHaveIgnoredAtRule(options, atRule)` so that it matches [the existing](https://github.com/stylelint/stylelint/blob/master/src/utils/optionsHaveIgnored.js) options utils. @davidtheclark What do you think?